frontend/src/metabase/search/components/SearchResult/components/Score.tsx

```diff
@@ -1,6 +0,0 @@
-import CS from "metabase/css/core/index.css";
-import type { SearchScore } from "metabase-types/api";
-
-export function Score({ scores }: { scores: SearchScore[] }) {
-  return <pre className={CS.hide}>{JSON.stringify(scores, null, 2)}</pre>;
```

Comment:
This component seems like overkill.. the develop can just look at the Network tab directly.